interfaces/builtin/libvirt.go

```diff
@@ -43,6 +43,7 @@ func NewLibvirtInterface() interfaces.Interface {
 		connectedPlugAppArmor: libvirtConnectedPlugAppArmor,
 		connectedPlugSecComp:  libvirtConnectedPlugSecComp,
 		reservedForOS:         true,
-		autoConnect:           true,
+		// cannot auto-connect, it grants too much power
+		autoConnect: false,
```

Comment:
Thanks for making this change. Can you add a test for autoconnect? Something like:

```
func (s *LibvirtInterfaceSuite) TestAutoConnect(c *C) {
	c.Check(s.iface.AutoConnect(), Equals, false)
}
```